src/main/scala/epic/dense/OutputEmbeddingTransform.scala

```diff
@@ -0,0 +1,163 @@
+package epic.dense
+
+import breeze.linalg._
+import breeze.linalg.operators.OpMulMatrix
+import epic.features.SegmentedIndex
+import epic.framework.Feature
+
+import scala.runtime.ScalaRunTime
+import scala.util.Random
+
+case class OutputEmbeddingTransform[FV](numOutputs: Int, outputDim: Int, innerTransform: Transform[FV, DenseVector[Double]], coarsenerForInitialization: Option[Int => Int] = None) extends OutputTransform[FV, DenseVector[Double]] {
```

Comment:
quick note about this too, please (pointer to relevant section of paper is fine.)